### PR TITLE
 replaces 💅 styled components with 👩‍🎤 emotion

### PR DESCRIPTION
**Pull request by clayton-duarte**
- Replaces 💅 styled-components with 👩‍🎤 emotion
- It will fix the following known issues:
https://styled-components.com/docs/faqs#why-am-i-getting-a-warning-about-several-instances-of-module-on-the-page
https://styled-components.com/docs/faqs#how-do-i-fix-flickering-text-after-server-side-rendering
- Emotion "just works in Next.js" as their documentation says:
https://emotion.sh/docs/ssr#nextjs

### CODE REVIEW

File: src/components/CroppedImage/index.mdx [hunk 1]
```diff
@@ -56,9 +56,9 @@ import Text from '../Text'
 <CroppedImage
   justify="center"
   height="150px"
-  radius="20px"
   width="150px"
   align="end"
+  radius
```
Comment (l0rdcafe) [inline src/components/CroppedImage/index.mdx:61]: question: just curious why this changed?

Comment (clayton-duarte) [reply src/components/CroppedImage/index.mdx:61]: I just realized that this prop is expecting a boolean, not a string... MDX is a little dumb and don't warn about types

File: src/index.tsx [hunk 2]
```diff
@@ -76,165 +67,154 @@ interface FluidRangeConfiguration {
...
+            50% {
+              left: 0px;
+            }
+            75% {
+              left: -5px;
+            }
+            100% {
+              left: 0px;
+            }
+          }
```
Comment (l0rdcafe) [inline src/index.tsx:206]: question: is this an SSR optimization?

Comment (clayton-duarte) [reply src/index.tsx:206]: well, kinda... I replaced Styled-Components by Emotion to fix a couple of issues we had and also to trim some configs (emotion works with Next.js out of the box). Most apis of those libs works in exactly the same way (like the styled method, the ThemeProvider and the serializer), but the global styles API is slightly different, that's why this guy looks different

**Review by l0rdcafe — APPROVED** [review]

**Comment by mitchied** [comment]
@clayton-duarte do i have to make any follow up tickets? If we're not using styled components then we'll have to clean up ark as well

**Review by mitchied — APPROVED** [review]

**Comment by clayton-duarte** [comment]
> @clayton-duarte do i have to make any follow up tickets? If we're not using styled components then we'll have to clean up ark as well

True, I'll update OOF and if everything is working I'll create that ticket... Thanks for the reminder

**Pull request merged**